docs/sources/shared/flow/reference/components/loki-server-http.md

```diff
@@ -0,0 +1,19 @@
+---
+aliases:
+- /docs/agent/shared/flow/reference/components/loki-server-http/
+headless: true
+---
+
+The `http` block configures the HTTP server.
+
+The following arguments can be used to configure the `http` block. Any omitted
+fields take their default values.
+
+ Name                   | Type       | Description                                                                                                          | Default  | Required
+------------------------|------------|----------------------------------------------------------------------------------------------------------------------|----------|----------
+ `listen_address`       | `string`   | Network address on which the server will listen for new connections. Defaults to accepting all incoming connections. | `""`     | no
```

Comment:
Is the default here `0.0.0.0`?

Reply:
Nop, is empty string which has the same effect of listening to tcp conns from every incoming ip. [Explained here](https://pkg.go.dev/net#Listen)